packages/lit-html/src/private-ssr-support.ts

```diff
@@ -57,6 +57,14 @@ export const _$LH = {
         return resolveOverrideFn(this, values);
       }
     },
+  patchDirectiveResolve: (
+    directiveClass: new (part: PartInfo) => Directive,
+    resolveOverrideFn: Directive['_$resolve']
+  ) => {
+    if (directiveClass.prototype._$resolve !== resolveOverrideFn) {
+      directiveClass.prototype._$resolve = resolveOverrideFn;
```

Comment:
Maybe needed?
```suggestion
    // This property needs to remain unminified.
    if (directiveClass.prototype['_$resolve'] !== resolveOverrideFn) {
      directiveClass.prototype['_$resolve'] = resolveOverrideFn;
```

Reply:
I think _$resolve does need to be minified. It should be minified to `S`.

Reply:
oops you're totally right!